tests/unit/introspection.tcl

```diff
@@ -558,6 +558,7 @@ start_server {tags {"introspection"}} {
             socket-mark-id
             req-res-logfile
             client-default-resp
+            io_uring
```

Comment:
I think we should probably introduce a new test variation for io_uring that builds the server with io_uring and run it with io_uring enabled.